content/mondoo-linux-security.mql.yaml

```diff
@@ -723,6 +728,79 @@ queries:
 
             sysctl -w kernel.randomize_va_space=2
             ```
+  - uid: mondoo-linux-security-restrict-kernel-address-access
+    title: Restrict access to kernel addresses
+    impact: 70
+    mql : |
+      kernel.parameters["kernel.kptr_restrict"] == 2
+    docs:
+      desc: |
+        This check verifies that access to kernel addresses is restricted by confirming that the kernel parameter kernel.kptr_restrict is set to 2, which limits the exposure of kernel pointers in user-space interfaces.
```

Comment:
I think it is a great addition. We should also add the reference to https://access.redhat.com/articles/7133962 and also highlight, that this may affect existing tools.

Reply:
Let me add that to the followup I have going